cmd/object-handlers.go

```diff
@@ -48,16 +49,37 @@ func setGetRespHeaders(w http.ResponseWriter, reqParams url.Values) {
 	}
 }
 
+// getSourceIPAddress - get the source ip address of the request.
+func getSourceIPAddress(r *http.Request) string {
+	// Attempt to get ip from standard headers.
+	for _, h := range []string{"X-Forwarded-For", "X-Real-Ip"} {
```

Comment:
We need to look for r.RemoteAddr, i don't see if we need to look for these type of headers. it should be simple as 

```
        // Get host and port from Request.RemoteAddr.
        host, _, _ := net.SplitHostPort(r.RemoteAddr)
```